component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtilTest.java

```diff
@@ -601,4 +600,61 @@ public IObjectFactory getObjectFactory() {
 
         return new PowerMockObjectFactory();
     }
+
+    @Test(description = "Test case for getProcessedClaimValue()", dataProvider = "processedClaimValueTestDataProvider")
```

Comment:
Shall we comment on the entire test case, otherwise it misinterprets as the unit test always passed.